.github/workflows/schedule-load-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Schedule Load Test every other Friday
+
+on:
+  schedule:
+    - cron: "0 0 * * 5/2"
+  workflow_dispatch: # Supports manual triggering
+
+env:
+  # Written for the qa stage, but could be used for dev or prod (empty string in that case)
+  STAGE: "qa"
+  # locust parameters. Refer to https://docs.locust.io/en/stable/configuration.html for explanation
+  LOCUST_USERS: 100
+  LOCUST_RATE: 10
+  LOCUST_DURATION: 180
+
+jobs:
+  load-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install locust
+
+      - name: Get an access token
+        id: getAccessToken
+        run: |
+          set +e  # Do not exit if error. Handle error messages here.
+          REPLY=`curl --location 'https://api-qa.mobilitydatabase.org/v1/tokens' \
+             --header 'Content-Type: application/json' \
+             --data '{ "refresh_token": "${{ secrets.QA_API_TEST_REFRESH_TOKEN }}" }'`
+          if [ $? -ne 0 ]; then
+            echo "Error: Cannot obtain access token Reply = \"$REPLY\""
+            exit 1  # Terminate the workflow
+          fi
+          ACCESS_TOKEN=`echo $REPLY | jq -r .access_token`
+          
+          if [ $? -ne 0 ]; then
+            echo "Error: Cannot extract access token from reply \"$REPLY\""
+            exit 1
+          fi
+          
+          if [ -z "ACCESS_TOKEN" ]; then
+            echo "Error: Access token is empty extracted from $REPLY"
+            exit 1
+          fi
+          
+          echo "ACCESS_TOKEN=$ACCESS_TOKEN" >> $GITHUB_ENV
+
+      - name: Run the load tests
+        run: |
+          # For the URL, we want to use the STAGE variable to create api.mobilitydatabase.org, 
+          # api-qa.mobilitydatabase.org or api-dev.mobilitydatabase.org. If the STAGE var is empty, don't add '-' prefix
+          [ -n "$STAGE" ] && STAGE="-${STAGE}"   
+          
+          FEEDS_AUTH_TOKEN="${{ env.ACCESS_TOKEN }}"    # The locust script uses this variable
+          locust -f ./load-test/gtfs_user_test.py --host=https://api${STAGE}.mobilitydatabase.org \
+            -u ${LOCUST_USERS} -r ${LOCUST_RATE} --headless -t ${LOCUST_DURATION} --only-summary --csv locust_results
```

Comment:
I generally agree, except the crux of the script is really one line. The rest is just because I want to give a bit more info for failures. 
Maybe I can just condense the if's into one liners? e.g. 
```
[ -z "$ACCESS_TOKEN" ] && { echo "Error: Access token is empty extracted from $REPLY"; exit 1; }
```
that way it won't seem so much code?
Or just use the _normal_ mechanism of GH actions by removing the `set +e` at the beginning, with the downside that the reason for failures being a bit less clear.

Reply:
Changes the tests to one liners so it looks more compact.